src/frontend-scripts/components/section-right/Generalchat.jsx

```diff
@@ -88,6 +88,45 @@ export default class Generalchat extends React.Component {
 		}
 	};
 
+	generalChatStatus = () => {
+		const { userInfo } = this.props;
+		const { userName } = userInfo;
+		const user = Object.keys(this.props.userList || []).length ? this.props.userList.list.find(play => play.userName === userName) : undefined;

```

Comment:
This is um disgusting, and a mostly unnecessary way to do this. You can just find without the ternary.

Reply:
Should be fixed both in generalChatStatus and gameChatStatus